libvirt.go

```diff
@@ -128,6 +128,20 @@ const (
 	MigrateFlagRDMAPinAll
 )
 
+// UndefineFlags specifies options available when undefining a domain.
+type UndefineFlags uint32
+
+const (
+	// UndefineFlagManagedSave removes all domain managed save data.
+	UndefineFlagManagedSave = 1 << iota
```

Comment:
I think you need to add the type explicitly for these to not just be of type `int`.